controller/api/models.py

```diff
@@ -643,7 +643,8 @@ def create(self):
         kwargs = {'memory': self.release.config.memory,
                   'cpu': self.release.config.cpu,
                   'tags': self.release.config.tags,
-                  'values': self.release.config.values}
+                  'values': self.release.config.values,
+                  'aname': self.app.id}
```

Comment:
Ahh I see. Let's just leave it as it makes for easier merges upstream if/when that happens. This LGTM - just needs Deis approval and certification.